src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -59,16 +64,51 @@ function teardown() {
         return
     fi
     kill_daemons $dir
-    if [ $(stat -f -c '%T' .) == "btrfs" ]; then
+    if [ `uname` != FreeBSD ] && \
+       [ $(stat -f -c '%T' .) == "btrfs" ]; then
         rm -fr $dir/*/*db
         __teardown_btrfs $dir
     fi
-    grep " $(pwd)/$dir/" < /proc/mounts | while read mounted rest ; do
+    grep " $(pwd)/$dir/" < ${PROCDIR}/proc/mounts | while read mounted rest ; do
         umount $mounted
     done
     rm -fr $dir
 }
 
+function timeout_diagnose() {
```

Comment:
> Are you suggesting to take it out all the way?

@wjwithagen yes.

> Reason I left it in, is that I do run into these timeout problems that I woudl like to be diagnosed correctly.

but again, if it were the problems about the test itself, we should fix the test first.